ChangeLog.d/psa_curve25519_key_support.txt

```diff
@@ -0,0 +1,10 @@
+Requirement changes
```

Comment:
Isn't this a default behaviour change too? Since psa_import_key now expects (correctly) to be fed a little-endian curve25519 key instead of assuming its input is big-endian?

Reply:
That's arguable, but I think it's just one bug. Whether it's a bug or a default behavior change hinges on whether the current behavior was intended. I can tell you it wasn't intended (I wrote the offending code, the intent was to have a spec which complies with common standards and to have a subset of this spec working, and Montgomery curves weren't in this subset and if I'd paid a little more attention I would have arranged to return `NOT_SUPPORTED` when trying to import one since the code was only written with Weierstrass curves in mind). But the documentation didn't state any limitation and didn't state the applicable reference, so my intention doesn't really help anyone else.

In any case you're right that it isn't just about `psa_export_key`, it's about the import/export format.